### ISSUE TITLE
`num_layers > 1` not really supported

### ISSUE THREAD

**Issue by silky** [una-dinosauria/human-motion-prediction#14]
thanks heaps for putting this code online!

i've been playing around with this code, and when trying to add more layers, i get a size mismatch:

```
ValueError: Trying to share variable basic_rnn_seq2seq/rnn/gru_cell/gates/kernel, but specified shape (2048, 2048) and found shape (1060, 2048).
```

i believe that the problem is in `rnn_cell_extensions.LinearSpaceDecoderWrapper`

basically, it seems to conjure up some magic variables `proj_w_out` and `proj_b_out`. but the sizes are wrong because i've set more than one GRU cell.

i.e. 1060 = 1024 + my input size, and it expects (2048,2048) because numlayers == 2

i think it should be an easy fix, but i'm not entirely sure what it is because i'm not sure where `proj_w_out` comes from ...

**Comment by una-dinosauria** [comment]
What tf version are you using? It seems like there was a breaking change for MultiRNNCell in tf 1.2 (6th point in https://github.com/tensorflow/tensorflow/blob/r1.3/RELEASE.md#major-features-and-improvements-1)

**Comment by una-dinosauria** [comment]
I think I got the error. Please check the code from master and let me know if it works for you.

**Comment by silky** [comment]
nice spot and thanks for the very prompt fix! it works for me now. i was using tensorflow 1.2.1

( note: you accidentally introduced a bug by deleting the code for when `num_layers <= 1` :) )

https://github.com/una-dinosauria/human-motion-prediction/blob/master/src/seq2seq_model.py#L85

**Comment by una-dinosauria** [comment]
Woops! I think I've fixed it on 1b39a73.

**Comment by silky** [comment]
hahaha!

http://imgur.com/a/TEg57
